### PR TITLE
fix broken args handling

### PR DESCRIPTION
**Pull request by John-Colvin**
and some formatting, mostly just fixing the mix of tabs and spaces

### CODE REVIEW

File: source/app.d [hunk 1]
```diff
@@ -8,23 +8,23 @@ struct CmdOptions
...
 	private string scriptOnRestart;
-	private int max=-1;
-	private int minUptime=1000;
+	private int max = -1;
+	private int minUptime = 1000;
 
-    ///
-	public void parse(string[] _args)
+	///
+	public void parse(ref string[] _args)
```
Comment (John-Colvin) [inline source/app.d]: `getopt` condenses the array when it removes an arg, need `ref` so the shortening is visible in the caller

File: source/app.d [hunk 2]
```diff
@@ -104,39 +105,39 @@ void main(string[] _args)
...
 	while((options.max == -1 || (options.max-- > 0)) && canRestart)
 	{
 		log("Starting: '%s'", cmdline);
 		uptime.start();
 
-        auto pid = spawnProcess(cmdline, std.stdio.stdin, outStream, errStream,
-            null, Config.retainStdout);
-            
-        auto exitCode = wait(pid);
+		auto pid = spawnProcess(_args[1 .. $], std.stdio.stdin, outStream, errStream,
```
Comment (John-Colvin) [inline source/app.d]: `spawnProcess` either takes a single string which is the name of a process, or an array of strings that is the process (optionally) followed by args.

**Comment by Extrawurst** [comment]
Thank you John!!

**Comment by John-Colvin** [comment]
Btw I've been using this package in production for over 2 years now, it's done it's (admittedly small) task very well, so thanks :)

**Comment by Extrawurst** [comment]
@John-Colvin haha thanks, same here, small but solid and reliable

**Pull request merged**